xios_examples/context_test/context_def_mod.F90

```diff
@@ -0,0 +1,46 @@
+module context_def_mod
+
```

Comment:
a code comment explaining what this module is here for would be really helpful please